integration_tests/src/main/python/delta_lake_merge_common.py

```diff
@@ -44,7 +44,10 @@ def do_merge(spark, path):
         dest_table = spark_tmp_table_factory.get()
         read_delta_path(spark, path).createOrReplaceTempView(dest_table)
         return spark.sql(merge_sql.format(src_table=src_table, dest_table=dest_table)).collect()
-    with_cpu_session(setup_tables)
+    if conf == None :
```

Comment:
nit: 

1. generally it is preferred to do `conf is None` over `conf == None`
2. with_cpu_session already has conf={} as a default. Why not just do the same above and then we would not need this if/else at all?